docs/Installing.asciidoc

```diff
@@ -20,32 +20,67 @@ https://github.com/os-autoinst/openQA[official repository].
 
 == Installation
 
-The easiest way to install openQA is from packages. You can find openSUSE
-packages in OBS in the
+Take in mind that there are a lot of disruptive changes between openQA
```

Comment:
Better "Keep in mind that there can be disruptive changes between openQA versions,"